docs/data/base/components/number-input/number-input.md

```diff
@@ -168,3 +168,10 @@ In the following snippet, if <kbd>Shift</kbd> is held when clicking the incremen
 ```jsx
 <NumberInput min={0} step={1} shiftMultiplier={5} />
 ```
+
+### Adornments
+
+You can use the `startAdornment` and `endAdornment` props to add a prefix or suffix to a number input.
+A common use case for adornments is when the component receives a specific unit of measure (like weight or currency).
```

Comment:
```suggestion
A common use case for adornments is displaying a specific unit of measure, like weight or currency.
```

@samuelsycamore Do you think we could simplify this sentence like this?

Reply:
Here's how I'd phrase this:

```suggestion
Adornments can be useful for displaying units of measure, like weight or currency, alongside values.
```